### DISCUSSION TITLE
How to add pywal support to nnn

### DISCUSSION THREAD

**Discussion by Ay1tsMe** [jarun/nnn#1752] [Q&A]
Right now I want to use my pywal color scheme with nnn. Currently I'm launching nnn with the `nnn -C` flag which uses my terminal colour scheme instead nnn's colours. This works but I also would like to have nerd font icons but using this flag overrides it and uses the terminal colours without icons.

Is there a way to change the colour scheme of nnn with pywal hex colours. This way, I would have the colour scheme of my pywal while also having the nerd font icons. I don't think using xterm256 colours would work as the hex decimal colours might be too different to the xterm256 colour.

**Comment by N-R-K** [comment]
Changing the icon colors require editing [`icons.h` file](https://github.com/jarun/nnn/blob/5b05c8b9b1e6fe5549680465e05aa89f45d265df/src/icons.h#L137-L158) and recompiling.
